bin/middlewares/user_language.js

```diff
@@ -7,7 +7,14 @@ module.exports = function (languageConfig) {
   const supportedLanguages = languageConfig.supportedLanguages;
   return function (req, res, next) {
     const langHeaders = req.acceptsLanguages();
-    if (langHeaders) {
+
+    // If cookie "l" is set and corresponds to a supported language, use it
+    if (req.cookies && req.cookies.l) {
```

Comment:
It looks like `req.cookies` is always undefined. I suppose we need to enable the `cookie-parser` middleware as suggested by https://expressjs.com/en/4x/api.html#req.cookies

Reply:
Ah indeed sorry, I was confused by the fact that I couldn't read any cookie locally, I thought it was due to my local instance of Maps not "linked" to a local instance of Phoenix.
After installing and using cookie-parser in app.js, I think we have something correct.
I couldn't manage to actually test it with a real "l=..." cookie though, as I didn't figure out yet how to set such a cookie locally...
I'd like to know more though, if you have more information on how to do that it's welcome. Thanks!